### PR TITLE
Update CreateDecryptConfig to parse x509 certs in keys

### PR DESCRIPTION
**Pull request by lumjjb**
Allow passing in of certs with decryption keys to help runtimes ingest certs for PKCS7 decryption

Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

File: helpers/parse_helpers.go [hunk 1]
```diff
@@ -138,7 +155,9 @@ func processPrivateKeyFiles(keyFilesAndPwds []string) ([][]byte, [][]byte, [][]b
 			gpgSecretKeyRingFiles = append(gpgSecretKeyRingFiles, tmp)
 			gpgSecretKeyPasswords = append(gpgSecretKeyPasswords, password)
 		} else {
-			return nil, nil, nil, nil, fmt.Errorf("unidentified private key in file %s (password=%s)", keyfile, string(password))
+			// ignore if file is not recognized, so as not to error if additinoal
```
Comment (stefanberger) [inline helpers/parse_helpers.go:158]: typo: additional

File: helpers/parse_helpers.go [hunk 2]
```diff
@@ -63,6 +63,23 @@ func processRecipientKeys(recipients []string) ([][]byte, [][]byte, [][]byte, er
 	return gpgRecipients, pubkeys, x509s, nil
 }
 
+// processx509Certs rpocesses x509 certificate files
```
Comment (stefanberger) [inline helpers/parse_helpers.go:66]: typo: process

**Comment by stefanberger** [comment]
otherwise LGTM

**Comment by lumjjb** [comment]
Fixed!

**Pull request merged**